### ISSUE TITLE
Steps towards submitting the product to GA4GH

### ISSUE THREAD

**Issue by andrewyatz** [ga4gh/refget#35]
These are the steps I believe are needed before we can submit to GA4GH product approval

- [x] Fill out the necessary forms from secretariat (Reggan to distribute)
- [x] Complete security and REWS review forms
- [x] Complete two implementations of the seqcol specification
- [x] Formalise what will be submitted i.e. what is the spec and what is additional items we are writing that side alongside the spec
- [x] Complete client(s) for the specification
- [x] Compliance checks/suite of tool. @yash-puligundla to advise

**Comment by andrewyatz** [comment]
Note these should be considered not in order

**Comment by yash-puligundla** [comment]
It would be nice to have the compliance suite output report follow [this schema](https://github.com/ga4gh/ga4gh-testbed-lib/tree/main/schema) so that it can be easily integrated into the testbed platform in the future.
I plan to move all the reusable components in the DRS compliance suite (https://github.com/ga4gh/drs-compliance-suite/tree/yp_add_test_cases_object_endpoint) to a common repo. But unfortunately, we are not there yet. In the meantime, I am available to discuss/ brainstorm so we are on the same page and are not duplicating any efforts.

**Comment by nsheff** [comment]
Hi @yash-puligundla can you give us an update on this? I think we're ready for some help with starting to implement a compliance suite at this point.

**Comment by yash-puligundla** [comment]
Hi @nsheff. As of now, we haven't proceeded with migrating the common code into a library.  
Could you please provide an estimate of the timeframe within which you would like this common code library to be available?

I'm also looping in @jppay, our new team lead, to this conversation.

**Comment by nsheff** [comment]
@andrewyatz can you talk about timeline here? I think we're wanting to move forward on this now, and I think we were targeting something in January, but I'm not exactly sure what that is. Can you give us a timeline and a bit more specifics on this?

**Comment by tcezard** [comment]
For information here are the documentation that we filled in for the RefGet v2 product approval:
 - Product approval questionnaire: https://docs.google.com/document/d/1p-zlQrhCCBHOo6BCmhuIjFWXJz9ScV2R/edit
 - REWS & DSWS: https://docs.google.com/document/d/1ZAhUJ2Ns0VCOiHmIt8Glr17w2COlvXrp/edit#heading=h.gjdgxs
 - Timeline: https://docs.google.com/document/d/1SpnyONLNta21ZVW97oiVYrYeujcnVGhR/edit#heading=h.gjdgxs
Note that the forms might have changed.

**Comment by tcezard** [comment]
All the tasks of this issue have been done except the Compliance checks/suite. It probably needs its own issue though. We should follow with #74
